### PR TITLE
Fe 45 demo user

### LINKED ISSUE #45: FE: Demo User
This ticket does not have a mock-up so please add in the button in the Login and Sign up pages and make sure the design is consistent with the overall look and feel.

- Create a "demo user" functionality that will be used later to allow recruiters and managers to easily test the app's features without having to sign up
- Add a button to allow for signing in as a demo user
- This is effectively a login button with a hardcoded username and password
- Show button both on signup and login pages
- In production you should manually create a user and some test data that would best show off your project. Do this in development right now to test this feature

### PR DESCRIPTION
**Pull request by jonahtabb**
### What this PR does (required):
- resolves #45 
- This creates a 'View Demo' button on both the sign up and login pages

### Screenshots / Videos (front-end only):
image
image

### Any information needed to test this feature (required):
- In order to test this feature you need to have a user already registered with the following credentials:
- username: demouser@demouser.com
- password: 12345678

### Any issues with the current functionality (optional):
- I could refactor the demo user function into a helper function to reduce duplicate code, but was unsure if that would be the best route.
- Perhaps the demo user credentials could be stored in an environment file instead of directly in the JSX, but I don't have a clear idea of how environment variables work in a react-typescript project.

### CODE REVIEW

**Review by ezeilo-su — COMMENTED** [review]
Looks good! Please you should always link your ticket to your PRs as it's a requirement here :smiley:  Check this [guide](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)

**Comment by rajivtitus** [comment]
Good work here Jonah! Just a comment from my end, please have a look!

File: client/src/pages/Login/LoginForm/LoginForm.tsx [hunk 1]
```diff
@@ -91,6 +92,18 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
             <Button type="submit" size="large" variant="contained" color="primary" className={classes.submit}>
               {isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'Login'}
             </Button>
+            <Button
```
[the remark this answers is not in the document]
Comment (jonahtabb) [reply client/src/pages/Login/LoginForm/LoginForm.tsx:95]: Sounds great! I have componentized the Demo Login button.

Question: Should I also turn the 'Handle Demo' function into a helper function or new useContext?  Currently it is one level up in the 'Login' and 'SignUp' components (not the 'LoginForm' and 'SignupForm' components)?

[the remark this answers is not in the document]
Comment (jonahtabb) [reply client/src/pages/Login/LoginForm/LoginForm.tsx]: resolved by c003f0b2eda872c3b53eef0d1011a72786bff280

**Review by rajivtitus — APPROVED** [review]
Excellent work here 💯

**Pull request merged**